static/templates/settings/admin_user_group_list.hbs

```diff
@@ -15,6 +15,9 @@
             <button class="button undo-button btn-danger small">
                 <i class="fa fa-undo" aria-label="{{t 'Cancel' }}" title="{{t 'Cancel' }}"></i>
             </button>
```

Comment:
I don't think we need this `Undo` button now since we are using modal.

Reply:
```Undo``` button is still functional for group members. Shall we remove ```Undo``` functionality entirely from the user group settings?

Reply:
Made this change in the updated commits.